Test/metatests/OutputEncoding.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %baredafny verify %args "%s" > "%t"
+// RUN: %baredafny run --no-verify --target=cs %args "%s" >> "%t"
+// RUN: %baredafny run --no-verify --target=js %args  "%s" >> "%t"
+// RUN: %baredafny run --no-verify --target=go %args  "%s" >> "%t"
+// RUN: %baredafny run --no-verify --target=py %args  "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
```

Comment:
Missing the `--target=java` case.

I'm just fine with ignoring C++ for now, but could we use `%testDafnyForEachCompiler` and just add an `int` somewhere (so C++ can cleanly opt out of this test case)?

Reply:
This is on purpose, because Java doesn't work.
The next patch will make it work (by forcing UTF-8)

Reply:
Never mind, reading comprehension fail on the PR description. :)